.travis.yml

```diff
@@ -13,3 +13,7 @@ script:
 after_success:
   - coverage report
   - coveralls
+  - git config --global user.name "semantic-release (via TravisCI)"
```

Comment:
```suggestion
  - git config --global user.name "ladybugbot"
  - git config --global user.email "ladybugbot@whateveremailprovider.com"
```

Reply:
I'd recommend using the ladybugbot user here as that is what will set the user releasing code. The GH_TOKEN is just a way to authorize automation.

Reply:
@AntoineDao, Thank you for the comments. See the updated version!

Reply:
Nice one!